src/dev-app/BUILD.bazel

```diff
@@ -121,6 +121,11 @@ create_system_config(
     output_name = "system-config.js",
 )
 
+filegroup(
```

Comment:
Could you add a comment that explains what this target is for / how it's used?

Reply:
Done.